icu4j/main/classes/core/src/com/ibm/icu/number/NumberFormatterImpl.java

```diff
@@ -36,6 +38,9 @@
 import com.ibm.icu.util.Currency;
 import com.ibm.icu.util.MeasureUnit;
 
+import java.util.Collections;
+import java.util.List;
+
```

Comment:
"The sort order of import statements is `java` / `javax` / `org` / `com`."

Reply:
done